packages/grant-explorer/src/features/common/Dropdown.tsx

```diff
@@ -0,0 +1,58 @@
+import { Menu, Transition } from "@headlessui/react";
+import {
+  Children,
+  cloneElement,
+  Fragment,
+  PropsWithChildren,
+  ReactElement,
+} from "react";
+import { ChevronDownIcon } from "@heroicons/react/20/solid";
+import tw from "tailwind-styled-components";
+
+export function Dropdown({
+  label,
+  children,
+}: { label?: string } & PropsWithChildren) {
+  return (
+    <Menu as="div" className="md:relative inline-block text-left z-20">
+      <div>
+        <Menu.Button className="inline-flex gap-2 items-center">
+          <span className="text-white py-2">{label}</span>
```

Comment:
if label is undefined, this will render as `<span>undefined</span>`. consider either making span a required prop, or making this whole block of JSX optional based on label.

Reply:
https://legacy.reactjs.org/docs/jsx-in-depth.html#booleans-null-and-undefined-are-ignored

Reply:
my bad - closing.